packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -338,6 +338,27 @@ test('limit', async t => {
   // t.throwsAsync(() => await E(seat1).getOfferResult());
```

Comment:
Is "TODO Offer result should be an error" something we should have for PSO?

Same question for the other TODO in this test: https://github.com/Agoric/agoric-sdk/blob/a1dedeae72908fda45afcb6038d76f8359adc8de/packages/inter-protocol/test/psm/test-psm.js#L328

Reply:
that's pre-existing. I don't know what was intended here.